### PR TITLE
Team Section edited

### PR DESCRIPTION
**Pull request by nimbus98**
Preview:

### CODE REVIEW

**Comment by anumehaagrawal** [comment]
@nimbus98  Have you added just core member details as of now ?

**Comment by nimbus98** [comment]
yes, the previous final year data is still present, and I'll just transfer them to a new section if the Alumni Section idea is accepted

**Comment by nimbus98** [comment]
Also solves #223

File: data/team.json [hunk 1]
```diff
@@ -1335,206 +1335,196 @@
...
-          "name": "Ambareesh",
-          "interests": "Machine Learning and Artificial Intelligence namely Markov Chain Monte Carlo methods, Deep Learning, Ensemble  Methods",
-          "email": "mailto:ambareesh.prakash@gmail.com",
-          "linkedin": "",
-          "github": "https://github.com/AeroDeath/",
-          "position": "Intelligence Group Head"
+          "id": "Suraj Jagannath",
+          "name": "Suraj Jagannath",
+          "interests": "Analog Electronics,Renewable Energy systems,Power Electronics",
+          "email": "mailto: suraj8730@gmail.com",
```
Comment (salman-bhai) [inline data/team.json]: Ideally mailto shouldn't have any space in between.

**Comment by anumehaagrawal** [comment]
@nimbus98 please make the changes

**Comment by anumehaagrawal** [comment]
@nimbus98 This has been pending for a while. Lets atleast update the team till we get a new website up.

**Pull request closed without merging**